clients/python/girder_client.py

```diff
@@ -423,3 +423,103 @@ def addMetadataToFolder(self, folderId, metadata):
         path = 'folder/' + folderId + '/metadata'
         obj = self.put(path, data=json.dumps(metadata))
         return obj
+
+    def downloadFile(self, fileId, path):
+        """
+        Download a file to the given local path.
+
+        :param fileId: The ID of the Girder file to download.
+        :param path: The local path to write the file to.
+        """
+        with open(path, 'wb') as fd:
+            req = requests.get('%s/file/%s/download' % (self.urlBase, fileId),
+                               headers={'Girder-Token': self.token})
+            for chunk in req.iter_content(chunk_size=65536):
+                fd.write(chunk)
+
+    def downloadItem(self, itemId, dest, name=None):
+        """
+        Download an item from Girder into a local folder. Each file in the
+        item will be placed into the directory specified by the dest parameter.
+        If the item contains multiple files or a single file with a different
+        name than the item, the item will be created as a directory under dest
+        and the files will become files within that directory.
+
+        :param itemId: The Id of the Girder item to download.
+        :param dest: The destination to write the item into.
```

Comment:
I'd change the description of `dest` to state that it is the destination *directory*.